Formula/srtp.rb

```diff
@@ -22,9 +22,13 @@ class Srtp < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
 
   def install
-    system "./configure", "--disable-debug", "--prefix=#{prefix}"
+    ENV.prepend "LDFLAGS", "-L#{Formula["openssl@1.1"].opt_lib}"
+    ENV.prepend "CPPFLAGS", "-I#{Formula["openssl@1.1"].opt_include}"
+
+    system "./configure", "--disable-debug", "--prefix=#{prefix}", "--enable-openssl"
```

Comment:
Why do we want to enable OpenSSL?

Reply:
I was trying to use `libsrtp` with `janus-gateway`, and it only works if the library is compiled with openssl. I was considering finding a way to make this optional, but I'm having trouble finding an example in another formula.